testing/test_recwarn.py

```diff
@@ -284,3 +284,27 @@ def test(run):
         ''')
         result = testdir.runpytest()
         result.stdout.fnmatch_lines(['*2 passed in*'])
+
+    def test_message_using(self):
+        source = "warnings.warn('my runtime warning', RuntimeWarning)"
+        pytest.warns(RuntimeWarning, source, message='my runtime warning')
```

Comment:
I've changed `test_message_using` in favor of `test_message`. (The using was a copy-paste error)